src/libraries/Common/src/System/Security/Cryptography/AsymmetricAlgorithmHelpers.Der.cs

```diff
@@ -0,0 +1,72 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics;
+
+namespace System.Security.Cryptography
+{
+    internal static partial class AsymmetricAlgorithmHelpers
```

Comment:
Is this accidentally added to this PR? It looks like EC-DSA related

Reply:
couldn't you use `ECDsa.GetMaxSignatureSize(DSASignatureFormat)` is it because we need this in M.B.C?

Reply:
Yep, it's split into a partial for M.B.C.